validator/src/commands/wait_for_restart_window/mod.rs

```diff
@@ -21,8 +24,31 @@ use {
     },
 };
 
+const COMMAND: &str = "wait-for-restart-window";
+
+#[derive(Debug, PartialEq)]
+pub struct WaitForRestartWindowArg {
```

Comment:
Let's make this `WaitForRestartWindowArgs` like we did in the other PR; there are indeed multiple args for this command

Reply:
[538f7d6](https://github.com/anza-xyz/agave/pull/4902/commits/538f7d65cee854134246e1608911041b1224e949) sorry for missing this one 🫠 will update to all PRs as well